src/fprime_gds/common/testing_fw/pytest_integration.py

```diff
@@ -52,6 +52,12 @@ def pytest_addoption(parser):
         action="store_true",
         help="Enable JUnitXML report generation to a specified location"
     )
+    # Add an option to specify a json config file that maps instances
+    parser.addoption(
+        "--deployment-config",
```

Comment:
This functionality is great - we should document it in https://github.com/nasa/fprime/blob/devel/docs/user-manual/gds/gds-test-api-guide.md